devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
IIRC CONF= was supposed to set BUILDTYPE? (see comments on top of devel/Makefile)
In particular we want to loadtest OptDebug, not pure debug.